Lagt til skedulert metode for å sette status på alle søknader under a…

nils-arne opened the pull request:
…rbeid til AUTOMATISK_SLETTET.

nils-arne committed:
Lagt til skedulert metode for å sette status på alle søknader under arbeid til AUTOMATISK_SLETTET.

enstulen commented on sendsoknad-business/src/main/java/no/nav/sbl/dialogarena/soknadinnsending/business/batch/SlettAktiveSoknaderScheduler.java:
```diff
@@ -0,0 +1,52 @@
+import static org.slf4j.LoggerFactory.getLogger;
+
+@Service
+@EnableSchedulerLock(defaultLockAtMostFor = "15m")
```
Sette denne til 
```suggestion
@EnableSchedulerLock(defaultLockAtMostFor = "20m")
```

nils-arne replied:
Oppdatert

enstulen commented on sendsoknad-business/src/main/java/no/nav/sbl/dialogarena/soknadinnsending/business/batch/SlettAktiveSoknaderScheduler.java:
```diff
@@ -0,0 +1,52 @@
+    @Scheduled(cron = SCHEDULE_TIME)
+    @SchedulerLock(name = "slettAktiveSoknader", lockAtLeastFor = "15m")
+    public void slettGamleSoknader() {
+        if (Boolean.parseBoolean(System.getProperty("sendsoknad.batch.enabled", "true"))) {
```
Fjern?

nils-arne replied:
Fjernet

enstulen commented on sendsoknad-business/src/main/java/no/nav/sbl/dialogarena/soknadinnsending/business/batch/SlettAktiveSoknaderScheduler.java:
```diff
@@ -0,0 +1,52 @@
+    public void slettGamleSoknader() {
+        if (Boolean.parseBoolean(System.getProperty("sendsoknad.batch.enabled", "true"))) {
+            long startTime = System.currentTimeMillis();
+            logger.info("Starter jobb for å slette alle aaktive søknader");
```
```suggestion
            logger.info("Starter jobb for å slette alle aktive søknader");
```

nils-arne replied:
Rettet

nils-arne committed:
Oppatering etter PR

nils-arne committed:
Lagt til skedulert metode for å sette status på alle søknader under arbeid til AUTOMATISK_SLETTET.
Midlertidig endret starttidspunkt for å kunne teste i dev.

nils-arne committed:
Lagt til skedulert metode for å sette status på alle søknader under arbeid til AUTOMATISK_SLETTET.
Midlertidig endret starttidspunkt for å kunne teste i dev. Fikset manglende konfigurering

nils-arne committed:
Lagt til skedulert metode for å sette status på alle søknader under arbeid til AUTOMATISK_SLETTET.
Midlertidig endret starttidspunkt for å kunne teste i dev. Fikset manglende konfigurering

nils-arne committed:
Lagt til skedulert metode for å sette status på alle søknader under arbeid til AUTOMATISK_SLETTET.
Midlertidig endret starttidspunkt for å kunne teste i dev. Fikset manglende konfigurering

nils-arne committed:
Lagt til skedulert metode for å sette status på alle søknader under arbeid til AUTOMATISK_SLETTET.
Midlertidig endret starttidspunkt for å kunne teste i dev. Fikset manglende konfigurering

nils-arne committed:
Lagt til skedulert metode for å sette status på alle søknader under arbeid til AUTOMATISK_SLETTET.
Midlertidig endret starttidspunkt for å kunne teste i dev. Fikset manglende konfigurering

nils-arne committed:
Lagt til skedulert metode for å sette status på alle søknader under arbeid til AUTOMATISK_SLETTET.
Midlertidig endret starttidspunkt for å kunne teste i dev. Fikset manglende konfigurering

nils-arne committed:
Lagt til skedulert metode for å sette status på alle søknader under arbeid til AUTOMATISK_SLETTET.
Midlertidig endret starttidspunkt for å kunne teste i dev. Fikset manglende konfigurering

nils-arne committed:
Lagt til skedulert metode for å sette status på alle søknader under arbeid til AUTOMATISK_SLETTET.
Midlertidig endret starttidspunkt for å kunne teste i dev. Fikset manglende konfigurering

nils-arne committed:
Lagt til skedulert metode for å sette status på alle søknader under arbeid til AUTOMATISK_SLETTET.
Lagt til redirect respons ved kall for oppretting av søknad

nils-arne committed:
Lagt til skedulert metode for å sette status på alle søknader under arbeid til AUTOMATISK_SLETTET.
